web/geo/types.ts

```diff
@@ -11,6 +11,7 @@ export interface ZoneConfig {
   contributors?: string[];
   disclaimer?: string;
   estimation_method?: string;
+  generation_only?: boolean;
```

Comment:
This relates to the other comment, if we exclude the zones all together we don't need this.